x-pack/plugins/apm/ftr_e2e/cypress/e2e/power_user/integration_settings/integration_policy.cy.ts

```diff
@@ -60,7 +60,7 @@ describe('when navigating to integration page', () => {
     cy.visitKibana(integrationsPath);
 
     // open integration policy form
-    cy.getByTestSubj('integration-card:epr:apm:featured').click();
+    cy.getByTestSubj('integration-card:ui_link:apm').click();
```

Comment:
I have changed this link as `integration-card:epr:apm:featured` will be removed very soon.